core/enums/error_code.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python3

```

Comment:
Please split this PR into multiple commits. One with existing files taken from Avalon repo where you can put original author(s) and you as the committer.

Reply:
Please check https://github.com/hyperledger-labs/avalon-sdk-py/pull/4
This has all the files taken from Avalon repo with all commit history and original author details.